docs/vgclog_examples.md

```diff
@@ -706,3 +707,910 @@ Analyzing the structure and elements of this example log output shows that this
+
+#### Sweep phase
+
+The sweep operation has two objectives:
+
+- To directly reclaim some memory by creating empty regions.
+- To build information about occupancy and fragmentation for regions that still contain live objects. The next partial GC cycle uses this information to defragment older regions.
+
+While the global *sweep* operation is logically associated with the global *mark* phase, it does not run in the same global mark cycle. Instead, the sweep operation runs in the same STW increment as the first partial GC cycle that runs after the completion of the global mark cycle. This can be seen in the following log excerpt.  After the log records the end of the global mark cycle it records a STW pause followed by a `partial gc` cycle of `id=1229`. The global sweep operation that runs after the the global mark phase is recorded in the `<gc-op>` element tagged as `id=1229`.
+
+```xml
+<exclusive-start id="1227" timestamp="2021-02-26T11:17:38.804" intervalms="1940.125">
+...
+<cycle-start id="1229" type="partial gc" contextid="0" timestamp="2021-02-26T11:17:38.805" intervalms="3926.202" />
+...
+</gc-start>
+...
+</gc-start>
+<gc-op id="1232" type="sweep" timems="9.472" contextid="1229" timestamp="2021-02-26T11:17:38.815" />
+<gc-op id="1233" type="copy forward" timems="308.258" contextid="1229" timestamp="2021-02-26T11:17:39.124">
+...
+<gc-op id="1234" type="classunload" timems="0.012" contextid="1229" timestamp="2021-02-26T11:17:39.125">
+...
+<gc-end>
+...
+</gc-end>
+<cycle-end id="1237" type="partial gc" contextid="1229" timestamp="2021-02-26T11:17:39.125" />
+<exclusive-end id="1238" timestamp="2021-02-26T11:17:39.125" durationms="320.792" />
+```
+
+A record of object liveness is now complete.
+*
+#### Summary of the  `balanced` global mark GC cycle example
+
+Analyzing the structure and elements of this example log output shows that this example `balanced` global mark GC cycle has the following characteristics:
+
+- If the total free memory is low when the taxation allocation threshold is reached, the GC triggers a global mark cycle. The allocation taxation threshold is set by the previous cycle to trigger a new cycle when the eden space is half full. This threshold value frees up eden space to enable a global mark cycle to interleave with the garbage collection operations of partial GC cycles.
+- Each global mark phase increment is triggered by an allocation taxation threshold value that is set to half of the eden space.
+- Global mark GC cycle and global mark cycle increments begin with a STW pause.
+- The global *mark* cycle does not reclaim memory. The cycle creates an updated record of object liveness by attempting to rebuild overflowed and stable regions. The change in status of the remembered set metastructure can be seen in the logs by inspecting the `<remembered-set>` attributes.
```

Comment:
It does 2 things independent things : 
- rebuilds object liveness information (rebuilds mark map)
- rebuilds remembered sets for overflowed and stable regions

An above sentenced contains 'by' which makes them dependent. Should be 'and'.

Reply:
Updated to: 

- The global *mark* cycle does not reclaim memory. The cycle creates an updated record of object liveness by rebuilding the mark map, and also attempts to rebuild the rememebered set for overflowed and stable regions. The change in status of the remembered set metastructure can be seen in the logs by inspecting the `<remembered-set>` attributes.